core/src/main/java/org/sonarsource/sonarlint/core/rules/OthersSectionHtmlContent.java

```diff
@@ -0,0 +1,50 @@
+/*
+ * SonarLint Core - Implementation
+ * Copyright (C) 2016-2022 SonarSource SA
+ * mailto:info AT sonarsource DOT com
+ *
+ * This program is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 3 of the License, or (at your option) any later version.
+ *
+ * This program is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ *
+ * You should have received a copy of the GNU Lesser General Public License
+ * along with this program; if not, write to the Free Software Foundation,
+ * Inc., 51 Franklin Street, Fifth Floor, Boston, MA  02110-1301, USA.
+ */
+
+package org.sonarsource.sonarlint.core.rules;
+
+import java.io.IOException;
+import java.nio.charset.StandardCharsets;
+import org.apache.commons.io.IOUtils;
+import org.sonarsource.sonarlint.core.commons.log.SonarLintLogger;
+
+public class OthersSectionHtmlContent {
+
+  private static final String FOLDER_NAME = "/context-rule-description/";
+  private static final String FILE_EXTENSION = ".html";
+  private static final String UNSUPPORTED_RULE_DESCRIPTION_FOR_CONTEXT_KEY = "Unsupported rule description for context key: ";
+  private static final String ERROR_READING_FILE_CONTENT = "Could not read the content for rule description for context key: ";
+  private OthersSectionHtmlContent() {}
+
+  public static String getHtmlContent(String contextKey) {
```

Comment:
If I'm correct, we only have one possible value for this method parameter? Why not simply changing to:
```suggestion
  public static String getHtmlContent() {
```